spinnaker/provider_test.go

```diff
@@ -1,19 +1,20 @@
 package spinnaker
 
 import (
+	"context"
 	"os"
 	"testing"
 
-	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
-	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 )
 
-var testAccProviders map[string]terraform.ResourceProvider
+var testAccProviders map[string]*schema.Provider
```

Comment:
[note]
[Removal of the terraform.ResourceProvider Interface](https://www.terraform.io/docs/extend/guides/v2-upgrade-guide.html#removal-of-the-terraform-resourceprovider-interface)

> The terraform.ResourceProvider interface has been removed. The concrete *helper/schema.Provider type should be used in its place.